extensions/nba-game-viewer/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # NBA Game Viewer Changelog
 
+## [Improvements] - {PR_MERGE_DATE}
+
+- Add the ability to select either the NBA or WNBA league to view games, standings, and more.
```

Comment:
```suggestion
- Adds the ability to select either the NBA or WNBA league to view games, standings, and more.
- Adds a preference to store last values for league, conference etc. in drop-downs instead of default values.
```

Reply:
@jamtur01 can you please apply this as well?